### DISCUSSION TITLE
Does Cypress version > 12.17.4 will ever be supported in the future ?

### DISCUSSION THREAD

**Discussion by samixchoumi** [sorry-cypress/sorry-cypress#911] [General]
Hello, regarding this : https://currents.dev/readme/integration-with-cypress/compatibility, do you know if Cypress will be supported on a version superior to `12.17.4` in the future ?
For the moment, new Cypress version don't give us big change that make us want to upgrade Cypress from v`12.17.4`, but if one day they do that, should we have the possibility to upgrade and still use `sorry-cypress` ?

**Comment by twk3** [comment]
The situation could always change, but at the moment it doesn't really seem feasible.

Because cypress clearly wants to prevent the use of it with anything associated with sorry-cypress in our package.json ([blog post on the matter](https://currents.dev/posts/v13-blocking)) any moves here will likely just be a whack-a-mole.

- sorry-cypress users could have renamed/forks of cypress-cloud to avoid the current blocking
  * They will likely just start blocking the new ones
- sorry-cypress and Currents and others could continue repacking the MIT licensed cypress runner without the blocking
  * But they could just take further action and change the license
 - Someone could fork the cypress runner
   * But the whole reason you're asking the questions is to get some of those new changes from cypress. 

To be clear, I think the blocking behaviour from cypress is in poor taste, and not aligned with what I would expect within a MIT licensed project that has a developer community. But this is how they have chosen to act.

**Reply by samixchoumi** [reply]
Thanks you for your answer, I'm thinking of switching to Playwright because of this but wanted to have more news about it. (I prefer being up-to-date on my project ahah)

**Comment by rrauenza** [comment]
I had myself considered writing a python script that just trawls through the cypress logs and pushes the results to sorry-cypress "out of band" so to speak.   I don't see how cypress could prevent that with technology, but they could change the license I suppose.

For us, I think we will just not write any new tests in Cypress and will look at playwright.
